client/src/main/java/io/pravega/client/auth/PravegaAuthHandler.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.auth;
```

Comment:
Just curious why this interface ended up in the client package?   I know it was moved there intentionally but I'm not clear why.

Reply:
This is for ease of implementation of custom plugins. Just for a single interface users need not have dependency on a completely new maven artifact (controller). They can just use the client artifacts to implement this.

Reply:
got it, thanks.